python/ql/src/experimental/semmle/python/Concepts.qll

```diff
@@ -163,7 +163,7 @@ class NoSQLQuery extends DataFlow::Node {
 
 module NoSQLSanitizer {
   abstract class Range extends DataFlow::Node {
-    abstract DataFlow::Node getSanitizerNode();
+    abstract DataFlow::Node getAnInput();
```

Comment:
Better name!

Reply:
Not my idea 😆 https://github.com/github/codeql/pull/5443#pullrequestreview-654178832